package.json

```diff
@@ -9,7 +9,7 @@
     "pretest": "yarn build",
     "lint": "eslint . --ext .ts",
     "format": "prettier --write 'packages/**/*.ts'",
-    "test": "jest",
+    "test": "tsc -b packages/dtslint && jest",
```

Comment:
This should already be covered by `"pretest"` 🤔

Reply:
Hmm, makes me think that dtslint isn't correctly integrated into the project build.

Reply:
Yep, that was it.